cloud_edition/flexibility_mode/docs/disk_usage.rst

```diff
@@ -0,0 +1,95 @@
+  
+- a file is uploaded using SFTP / scp / ...
+  
+Impact of Disk Usage on the PIM
+-------------------------------
+When the disk is full, the PIM cannot work anymore: it cannot save modifications in database, upload new asset, ...
+
+It could can even prevent ElasticSearch from updating its indexes, resulting in divergence with the database.
+To avoid those issues, ensuring a healthy disk usage is necessary.
+
+Improving Disk Usage
+--------------------
+To reduce disk usage, some temporary files are deleted automatically on a regular basis. For example:
+
+- old temporary import/export files
+- old temporary file storage
+- akeneo_batch directories
+- ansible temporary directories
+
+Moreover, the integrator can:
+
+- activate the purge of old versions of products
+- remove assets that are not linked to any product anymore
+
+.. code-block:: bash
+
+    mkdir -p /root/purge
+    
+    echo "Track assets that will be deleted in a csv file"
+    mysql akeneo_pim -e "SELECT fi.id, fi.original_filename, fi.file_key, r.file_info_id, r.asset_id, v.* FROM akeneo_file_storage_file_info fi LEFT JOIN pimee_product_asset_reference r ON fi.id = r.file_info_id LEFT JOIN pimee_product_asset_variation v ON fi.id = v.file_info_id WHERE storage = 'assetStorage' AND r.file_info_id IS NULL AND r.asset_id IS NULL AND v.source_file_info_id IS NULL" > /root/purge/asset_rows_to_delete.csv
+  
+    echo "List files to be deleted in a txt file"
+    echo "Path of files to be deleted is calculated by concatenating "/home/akeneo/pim/app/file_storage/asset/" with value of "fi.file_key" from the MySQL resquest."
+    mysql akeneo_pim -se "SELECT concat('/home/akeneo/pim/app/file_storage/asset/',fi.file_key) FROM akeneo_file_storage_file_info fi LEFT JOIN pimee_product_asset_reference r ON fi.id = r.file_info_id LEFT JOIN pimee_product_asset_variation v ON fi.id = v.file_info_id WHERE storage = 'assetStorage' AND r.file_info_id IS NULL AND r.asset_id IS NULL AND v.source_file_info_id IS NULL" > /root/purge/asset_files_to_delete.txt
+  
+    echo "Delete rows in database"
+    mysql akeneo_pim -se "SET FOREIGN_KEY_CHECKS=0;DELETE v, r, fi FROM akeneo_file_storage_file_info fi LEFT JOIN pimee_product_asset_reference r ON fi.id = r.file_info_id LEFT JOIN pimee_product_asset_variation v ON fi.id = v.file_info_id WHERE storage = 'assetStorage' AND r.file_info_id IS NULL AND r.asset_id IS NULL AND v.source_file_info_id IS NULL;SET FOREIGN_KEY_CHECKS=1;" > /root/purge/asset_rows_deleted.csv
+  
+    echo "Delete assets files based on previous list"
+    cat /root/purge/asset_files_to_delete.txt | xargs rm -f
+    
+
+- purge old versions from versioning table
+
+.. code-block:: bash
+
+    mkdir -p /root/purge
+    
+    echo "Cleansing versions older than 90 days"
+    nohup php -d memory_limit=-1 bin/console --env=prod pim:versioning:purge --more-than-days 90 --force -n &
+    
+    echo "Shrink Mysql tables"
+    nohup mysqlcheck -o akeneo_pim &
```

Comment:
no way ! 
with this command the partner will learn something... and we can have some strange behavior on the pim side due to lock.
i prefer use the mysql client and with optimize <table>. and we must explain "mysql will re-create a new file for <table>  and for that we need disk space... etc..."